lib/src/main/kotlin/trees/balancedTree.kt

```diff
@@ -0,0 +1,21 @@
+package trees
+
+import nodes.abstractNode
+
+abstract class balancedTree<K: Comparable<K>, V, someNode: abstractNode<K, V, someNode>>: abstractTree<K, V, someNode>() {
```

Comment:
Та же история про [название](https://kotlinlang.org/docs/coding-conventions.html#naming-rules) класса. 
```suggestion
abstract class BalancedTree<K: Comparable<K>, V, someNode: abstractNode<K, V, someNode>>: abstractTree<K, V, someNode>() {
```